homework-pages/calculatorStartPage.js

```diff
@@ -0,0 +1,40 @@
+const url = 'https://testsheepnz.github.io/BasicCalculator';
+const selectBuild = '#selectBuild';
+const selectBuildLabel = '#selectBuild';
+const field1 = '#number1Field';
```

Comment:
Name 'field1' is too generic. It should be very clear without need to read a code around. Maybe 'firstNumberInput'?